sigs/cncf-sigs.md

```diff
@@ -0,0 +1,252 @@
+
+* May not create the impression that they have any authority or formal responsibilities in the SIG other than assigned roles.
+
+### Elections
+
+* The TOC nominates Chairs
+
+* Chairs are assigned following a 2/3 majority vote of the TOC
+
+* Terms last for 1 year but staggered such that at least 1 of the chairs is able to maintain continuity
+
+* The TOC and Chairs nominate Tech leads
+
+* Tech leads are assigned following a majority vote of the TOC and SIG Chairs
+
+* SIG Chairs and Tech Leads may be unassigned from the SIG at any time following a 2/3 majority vote of the TOC
+
+### Governance
+
+* All SIGs inherit and follow the CNCF TOC Operating Principles.
+
+* SIGs must have a documented governance process that encourages community participation and clear guidelines to avoid biased decision-making.  
+
+    * NOTE: aim here is to align with "minimal viable" model of the CNCF projects, and only have such governance as is needed, not anything too burdensome
+
+* They may grow a set of practices over time in the same way as an OSS Project, provided this is consistent with CNCF Operating Principles. 
+
+* As with CNCF Projects all exceptions and disputes are handled by TOC with CNCF Staff help
+
+### Budget & Resource
+
+* No formal systematic budget at this time, other than commitment of CNCF executive staff to provide named person as liaison point.
+
+* Just as CNCF Projects may have "help" offered by CNCF and may ask for things via the [ServiceDesk](https://github.com/cncf/servicedesk), the SIGs may do this.
+
+## Retirement
+
+* In the event that a SIG is unable to regularly establish quorum, or fulfill the responsibilities and/or regularly report to the TOC, the TOC will:
+
+    * Consider retiring the SIG after 3 months
+
+    * Must retire the SIG after 6 months
+
+* The TOC may, by means of a 2/3 majority vote, declare "no confidence" in the SIG.  In this event, the TOC may then vote to retire or reconstitute the SIG.
+
+## Initial SIGS
+
+To bootstrap the process, the TOC proposes the following SIGs, and projects assigned to each SIG. Clearly all of these SIG’s will not be fully-formed overnight or begin operating immediately, so the TOC itself will fulfill the duties of not-yet-formed SIG’s until they are.  We can however, fairly immediately, assign one voting member of the TOC as liason for each SIG, and prioritize the order of formation of the SIGs, starting immediately with the most pressing ones. 
+
+<table>
+  <tr>
+    <td>Name (to be finalised)</td>
+    <td>Area</td>
+    <td>Current CNCF Projects</td>
+  </tr>
+  <tr>
+    <td>Traffic</td>
+    <td>networking, service discovery, load balancing, service mesh, RPC, pubsub, etc.</td>
+    <td>Envoy, Linkerd, NATS, gRPC, CoreDNS, CNI</td>
+  </tr>
+  <tr>
+    <td>Observability</td>
+    <td>monitoring, logging, tracing, profiling, etc.
+</td>
+    <td>Prometheus, OpenTracing, Fluentd, Jaeger, Cortex, OpenMetrics, </td>
+  </tr>
+  <tr>
+    <td>Governance</td>
```

Comment:
There was some question on the [mailing list](https://lists.cncf.io/g/cncf-toc/topic/cncf_sigs_proposal/29603804?p=,,,20,0,0,0::recentpostdate%2Fsticky,,,20,2,0,29603804) about whether this is the appropriate name for this group.

Some key concerns:
- The word "governance" is often used to convey human processes of policy (e.g. how decisions are made, roles and responsibilities, etc.)
- The word "governance" is used earlier in this same document to describe how the SIGs should be managed
- The topics for the SIG and list of projects are more about the software used to implement security and privacy, along with ensuring compliance (auditing, etc)
- Some open source projects have a GOVERNANCE.md (or similarly named directory) to define project roles and decision-making process (examples: Node, cloudevents, SAFE, docker, k8s community)

Alternative suggestions that were made:
- Security
- Security & Policy
- Secure Access for Everyone
- Security & Compliance
- RegSec
- Security & Governance
- Oversight

Reply:
@izgeri Agreed.  All SIGs will have the opportunity to wordsmith their names as part of the chartering process.  These names are placeholders.  The ultimate names will need to be agreed upon by the SIG chairs and TOC.  I will note that there have been objections to all of the alternatives listed too.

Reply:
Is the chartering process defined anywhere? I can't seem to find it in this doc.

Reply:
@izgeri See the "SIG Charter" and "SIG Formation..." sections.